tests/e2e/utils/onboarding-wizard-utils.js

```diff
@@ -144,7 +144,7 @@ export function testTitle( { text, element = 'h1' } ) {
 }
 
 /**
- * Reset plugin configuration.
+ * Reset plugin configuration'.amp-settings-nav'
```

Comment:
I suppose this is a typo.

```suggestion
 * Reset plugin configuration.
```